server/recceiver/cfstore.py

```diff
@@ -110,7 +117,7 @@ def _startServiceWithLock(self):
             """
             self.client = ChannelFinderClient()
             try:
-                cf_properties = [cf_property["name"] for cf_property in self.client.getAllProperties()]
+                cf_properties = {cf_property["name"] for cf_property in self.client.getAllProperties()}
```

Comment:
I see that this is used as a `set` below; if you're making this change, maybe remove the `set()`s below?

Reply:
Also, you're creating a CFProperty data class here but here you're reading things as a dictionary. This commit is a bit confusing.